modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,67 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		property portal.acceptance = "true";
```

Comment:
We don't add portal.acceptance into FP3 cases in headless qa team, please help remove them, thanks.
`property portal.acceptance = "true";`